templates/cli/bin/service.twig

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+php /usr/local/code/src/Appwrite/Services/{{service.name | caseUcfirst}}.php $@
```

Comment:
We usually locate functional/app files (non-oop / classes) in the `app` directory and not the `src`. We would also name then in all lowercase + dashes.

Reply:
Done